x/rewards/keeper/iprpc.go

```diff
@@ -164,7 +164,7 @@ func (k Keeper) distributeIprpcRewards(ctx sdk.Context, iprpcReward types.IprpcR
 			UsedReward = UsedRewardTemp
 
 			// reward the provider
-			_, _, err := k.dualstakingKeeper.RewardProvidersAndDelegators(ctx, providerCU.Provider, specFund.Spec, providerIprpcReward, string(types.IprpcPoolName), false, false, false)
+			_, _, err := k.dualstakingKeeper.RewardProvidersAndDelegators(ctx, providerCU.Provider, specFund.Spec, providerIprpcReward, string(types.IprpcPoolName), false, false, false, "iprpc rewards")
 			if err != nil {
 				utils.LavaFormatError("failed to send iprpc rewards to provider", err, utils.LogAttr("provider", providerCU))
 			}
```

Comment:
the iprpc pool emissions event needs to include all the rewards total, instead of needing to track each payment and doing a sum

Reply:
done. see https://github.com/lavanet/lava/commit/93ab557e0b96a7d57b3bbf80dc59e744345bb9e5